src/AkismetSpamProtector.php

```diff
@@ -56,37 +64,30 @@ class AkismetSpamProtector implements SpamProtector
      * @var array
      */
     private $fieldMapping = array();
-    
-    /**
-     * Overridden API key
-     *
-     * @var string
-     */
-    protected static $_api_key = null;
-    
+
     /**
      * Set the API key
-     * 
+     *
      * @param string $key
      */
     public static function set_api_key($key)
     {
-        self::$_api_key = $key;
+        self::$api_key = $key;
     }
     
     /**
      * Get the API key
-     * 
+     *
      * @return string
      */
     protected static function get_api_key()
     {
-        if (self::$_api_key) {
-            return self::$_api_key;
+        if (self::$api_key) {
+            return self::$api_key;
```

Comment:
I think now that we've had a look at what the `protected static $_api_key` was for, that we can just get rid of it entirely.
This simplifies the code in places like this, where this `if` statement is now entirely unneeded.